tools/mjml.ts

```diff
@@ -64,11 +65,15 @@ app.route("/mjml/:id/:type").get(
   function (req, res, next) {
     Followup.findById(req.params.id)
       .populate("simulation")
-      .exec(function (err, followup: any) {
+      .exec(function (err, followup: FollowupInterface | null) {
         if (err) {
           return next(err)
         }
-        req.followup = followup
+
+        if (followup) {
+          req.followup = followup
+        }
```

Comment:
J'aurais plutôt tendance à mettre la logique inverse ici pour spécifier l'erreur :
```suggestion
     if (!followup) {
        return res.sendStatus(404)
      }
      req.followup = followup
```

Reply:
Hm, dans l'idéal c'est mieux je suis d'accord. En revanche le soucis c'est que ça casserait le comportement (avant cette PR on envoi pas de 404 si pas de followup). Comme c'est un middleware qui est utilisé par pas mal de route le risque est quand même grand (ou le besoin de verification/ réecriture). 

À la rigueur je peux changer pour 
```
if (!followup) {
  next()
}
```